config/optional/amp.theme.yml

```diff
@@ -0,0 +1 @@
+amptheme: thunder_amp
```

Comment:
hm, that's true. if you install amp module without thunder_amp then you will also have this setting. which will be useless